compiler/src/iree/compiler/Dialect/Stream/IR/test/tensor_folding.mlir

```diff
@@ -206,3 +206,36 @@ util.func private @ElideUnneededTensorClones(%arg0: !stream.resource<*>, %arg1:
   // CHECK: util.return %[[T1]]
   util.return %2 : f32
 }
+
+// -----
+
+#encoding = #iree_encoding.pad_encoding_layout<[0, 0]>
```

Comment:
I think it might be good to add a `NoFold` test case with non-identity padding layout as well or is this tested already somewhere?

Reply:
Good point, I tested it locally only. Let me add a few more tests.